### ISSUE TITLE
AGG Backend

### ISSUE THREAD

**Issue by lateshift** [alandefreitas/matplotplusplus#23]
Hey!

Just a quick "question/note" ... isn't AGG semi-maintained through mapnik ?

See:

https://github.com/mapnik/mapnik/wiki/MapnikRenderers

(the last changes in the repo are *cough* 2 years old though)

Thanks for the great work!

Cheers
Jens

**Comment by alandefreitas** [comment]
Hey!
I reached the conclusion that it's unmaintained because of what I read in:
https://github.com/ghaerr/agg-2.6
And it's not even 2 years old 😆

It seems like not even http://www.antigrain.com/ is fully working anymore.

He says: "The official AGG site remains on Sourceforge, but doesn't see much activity. With www.antigrain.com recently inoperational, I thought a GitHub repo was in order, as the project demos stopped building with the site down. The demos now build without the website.

I have created PDFs of all of Maxim's articles from the previous website and may put those up as well, as they greatly aid in understanding the library."

I had no idea of https://github.com/mapnik/mapnik/wiki/MapnikRenderers but it seems even older than what's in https://github.com/ghaerr/agg-2.6.
